docs/tutorial/pre-requisites-of-running-on-chain-spam-protected-chat2.md

```diff
@@ -16,96 +19,89 @@ If you already have an Ethereum account with sufficient ethers on the Sepolia te
     
     ![](https://i.imgur.com/zFduIV8.jpg)
 
-4. You should already see an account created. As you can see on top right, it should be pointing to Ethereum mainnet.
+4 The automatically created account points to the Ethereum Mainnet by default (dropdown menu in the top right corner).
     
     ![](https://i.imgur.com/gk3TWUd.jpg)
 
-5. You can use the same account for different networks. For Waku we need to connect to the Sepolia test network.
-6. You can switch to a test network by clicking on the drop down menu. Select Sepolia test network for Waku:
+   You can use the same account for different networks. To publish to the Waku Network, we need to connect to the Sepolia test network.
```

Comment:
> publish to the Waku Network

It feels like the object is missing, as in "publish _messages_ to the network".

> You can use... we need to connect

Sounds inconsistent, I'd use "you" in both cases.

Reply:
Made the necessary changes in https://github.com/waku-org/nwaku/pull/2206/commits/4d921d9426bd22664ee3cf19ea355fb1219fe890